Packs/Code42/Integrations/Code42/Code42.py

```diff
@@ -194,246 +208,294 @@ def build_query_payload(args):
 
 
 @logger
 def alert_get_command(client, args):
     code42_securityalert_context = []
-    alert = client.get_alert_details(args['id'])
+    alert = client.get_alert_details(args["id"])
     if alert:
         code42_context = map_to_code42_alert_context(alert)
         code42_securityalert_context.append(code42_context)
         readable_outputs = tableToMarkdown(
-            f'Code42 Security Alert Results',
+            f"Code42 Security Alert Results",
             code42_securityalert_context,
-            headers=SECURITY_ALERT_HEADERS
+            headers=SECURITY_ALERT_HEADERS,
         )
-        return readable_outputs, {'Code42.SecurityAlert': code42_securityalert_context}, alert
+        return readable_outputs, {"Code42.SecurityAlert": code42_securityalert_context}, alert
     else:
-        return 'No results found', {}, {}
+        return "No results found", {}, {}
 
 
 @logger
 def alert_resolve_command(client, args):
-    code42_securityalert_context = []
-    alert = client.resolve_alert(args['id'])
-    if alert:
-        # Retrieve new alert details
-        updated_alert = client.get_alert_details(args['id'])
-        if updated_alert:
-            code42_context = map_to_code42_alert_context(updated_alert)
-            code42_securityalert_context.append(code42_context)
-            readable_outputs = tableToMarkdown(
-                f'Code42 Security Alert Resolved',
-                code42_securityalert_context,
-                headers=SECURITY_ALERT_HEADERS
-            )
-            return readable_outputs, {'Code42.SecurityAlert': code42_securityalert_context}, updated_alert
-        else:
-            return 'Error retrieving updated alert', {}, {}
-    else:
-        return 'No results found', {}, {}
+    code42_security_alert_context = []
+    alert_id = client.resolve_alert(args["id"])
+
+    if not alert_id:
+        return "No results found", {}, {}
+
+    # Retrieve new alert details
+    alert_details = client.get_alert_details(alert_id)
+    if not alert_details:
+        return "Error retrieving updated alert", {}, {}
+
+    code42_context = map_to_code42_alert_context(alert_details)
+    code42_security_alert_context.append(code42_context)
+    readable_outputs = tableToMarkdown(
+        f"Code42 Security Alert Resolved",
+        code42_security_alert_context,
+        headers=SECURITY_ALERT_HEADERS,
+    )
+    return (
+        readable_outputs,
+        {"Code42.SecurityAlert": code42_security_alert_context},
+        alert_details,
+    )
 
 
 @logger
 def departingemployee_add_command(client, args):
     departure_epoch: Optional[int]
     # Convert date to epoch
-    if args.get('departuredate'):
+    departure_epoch = None
+    if args.get("departuredate"):
         try:
-            departure_epoch = int(time.mktime(time.strptime(args['departuredate'], '%Y-%m-%d')))
+            departure_epoch = int(time.mktime(time.strptime(args["departuredate"], "%Y-%m-%d")))
```

Comment:
you should not have to convert this this way anymore, new py42 just takes the YYYY-mm-dd date string